### ISSUE TITLE
Asking about testing result

### ISSUE THREAD

**Issue by codaibk** [JiahuiYu/generative_inpainting#106]
Hi, is the testing image which you use in this project, is inside training image?
Because you just take the image from celeba dataset which you trained with model to test your model Thats why the result look good, but if I test your model with new testing image, the result looks not good.
Example:
This is the result with your testing image which inside training dataset:
Inpainting_image
![image](https://user-images.githubusercontent.com/28720676/43474759-57c8b9bc-9526-11e8-92a4-0ed51bbb42a7.png)
Result:
![image](https://user-images.githubusercontent.com/28720676/43474814-76826f92-9526-11e8-8ea9-7d6743dbdebd.png)
But if i use image not inside your training dataset the result looks not good:
Inpaingin_Image:
![image](https://user-images.githubusercontent.com/28720676/43474876-aade643a-9526-11e8-997f-d94270ba167b.png)
Result:
![image](https://user-images.githubusercontent.com/28720676/43474925-cb162170-9526-11e8-93c8-4ac578f2311c.png)
Groundtruth:
![image](https://user-images.githubusercontent.com/28720676/43475228-a6487f90-9527-11e8-96e5-42ecf1c5ac24.png)

- For testing image, i take from this [link](http://mmlab.ie.cuhk.edu.hk/projects/CelebA.html)

**Comment by JiahuiYu** [comment]
As shown in [FAQ](
https://github.com/JiahuiYu/generative_inpainting#faq), We use the default training/validation data split from Places2 and CelebA. For CelebA, training/validation have no identity overlap.

For your examples above, how do you crop the image? Why the image is not aligned?

You could have a try of our [demo](
http://jiahuiyu.com/deepfill/) on CelebA-HQ, where there is 2000 validation images for you to try with any mask.

**Comment by codaibk** [comment]
Hi, sorry for cropping the image like that. But actually, thats cropped image just using for posting here to ask question. 
This is really aligned my image using in testing
- First, your provide testing, the result looks good:
Inpainting:
![output_inpaint](https://user-images.githubusercontent.com/28720676/43498086-ed53225a-9577-11e8-8436-4ae6432e31ca.png)
Mask: 
![center_mask_256](https://user-images.githubusercontent.com/28720676/43498090-f2be4fe4-9577-11e8-9e17-7c2efa6942a8.png)
Output:

![output_6](https://user-images.githubusercontent.com/28720676/43498114-0ec440fe-9578-11e8-8544-839ea6fd33e5.png)
- Second: My testing input, the result looks bad
Inpainting: 
![ori_256_inpaint](https://user-images.githubusercontent.com/28720676/43498140-32d0f5dc-9578-11e8-9a0e-1d1d1fa42a41.png)
Mask:
![center_mask_256](https://user-images.githubusercontent.com/28720676/43498146-38333f08-9578-11e8-9f32-9463dad0bb9c.png)

Output:
![output_4](https://user-images.githubusercontent.com/28720676/43498201-733f1a86-9578-11e8-8435-0d8bc1f8bb99.png)

![output_4](https://user-images.githubusercontent.com/28720676/43498152-400b5ca6-9578-11e8-8266-aeed1f9285eb.png)
- For demo with [HQ](http://jiahuiyu.com/deepfill/) which you mentioned, again in this online demo, you already provide the input testing from you. Not provide the function for user uploading  new testing. So it doesnt solve my question
Thanks

**Comment by JiahuiYu** [comment]
@codaibk The model of demo HQ and train/val split is also released. You can try whatever image you like. There is no need for users to upload since you can just try it with our released model. Also in this case, the inpainting model tries to hallucinate eye glasses, which is rather corner distribution in CelebA dataset. Hope it helps to your confusion.

**Comment by codaibk** [comment]
@JiahuiYu My orriginal image without eye glasses, so it doesnt mean thats issu for CelebA dataset. I dont need to try online demo HQ because there is no function for me to test with new image which is not inside your dataset. So it doesnt solve my question because i am doubt about your model which you said it can generate the image from inpainting_image, but for my testing case, it didnt give good result. 
This is my original image which give bad result (FYI, not only this image returns bad result, the other images also)
![ori_256](https://user-images.githubusercontent.com/28720676/43533583-be163eee-95e7-11e8-988d-e6896fa5771d.png)
